src/main/java/frc/robot/maps/RobotMap.java

```diff
@@ -30,21 +35,36 @@ private EncodedSpeedController getMock() {
     }
 
     public static class IntakeMap {
+        
         public SendableSpeedController intake() {
             return new MockSpeedController();
         }
+        
+
+        public SendableSpeedController singulator() {
+            return new MockSpeedController();
+        }
+
+        public SendableSpeedController pierre() {
+            return new MockSpeedController();
+        }
 
         public IDSolenoid deployIntake() {
             return new MockDSolenoid();
         }
+
     }
 
     public IntakeMap getIntakeMap() {
         return new IntakeMap();
     }
 
     public static class ShooterMap {
-        public SendableSpeedController shooterWheel() {
+        public SendableSpeedController shooterWheel1() {
```

Comment:
There shouldn't be any shooter changes in this PR. Seems like there might be some weird merge issue.